lib/emerald.rb

```diff
@@ -21,35 +21,46 @@ module Emerald
   class CLI < Thor
     class_option :beautify, :type => :boolean, :aliases => 'b'
 
+    def self.exit_on_failure?
```

Comment:
This is a setting for Thor. If you override this method and set it to true, then Thor exits with status code -1 instead of 0 if you raise a `Thor::Error`.